Mathlib/Data/Set/Card/Arithmetic.lean

```diff
@@ -0,0 +1,76 @@
+/-
+Copyright (c) 2025 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+
+import Mathlib.Data.Set.Card
+import Mathlib.SetTheory.Cardinal.Arithmetic
+
+/-!
+# Results using cardinal arithmetic
+
+This file contains results using cardinal arithmetic that are not in the main cardinal theory files.
+It has been separated out to not burden `Data.Set.Card` with extra imports.
+
+## Main results
+
+- `exists_union_disjoint_ncard_eq_of_even`: Given a set `s` with an even cardinality, there exist
+  disjoint sets `t` and `u` such that `t ∪ u = s` and `t.ncard = u.ncard`.
+- `exists_union_disjoint_cardinal_eq_iff` is the same, except using cardinal notation.
+-/
+
+variable {α : Type*}
+
+open scoped Finset
+
+theorem Finset.exists_disjoint_union_of_even_card [DecidableEq α] {s : Finset α} (he : Even #s) :
+    ∃ (t u : Finset α), t ∪ u = s ∧ Disjoint t u ∧ #t = #u :=
```

Comment:
Please add an `iff` version of this theorem to mirror the one for `Set`.

Reply:
Done:)